src/ctia/bundle/core.clj

```diff
@@ -166,23 +166,18 @@
                              first
                              :entity
                              with-long-id
-                             ent/un-store)
-          num-old-entities (count old-entities)]
+                             ent/un-store)]
+      (when (< (count old-entities) 1)
```

Comment:
`(< (count old-entities) 1)` means `(zero? (count old-entities))`, which I'm not sure is the right thing?

Reply:
That's a bug, thx!
I am punished for not testing that the logging is actually done :-).